client/src/components/LinkButton.tsx

```diff
@@ -31,7 +29,7 @@ interface Props {
 export default function LinkButton(props: Props) {
```

Comment:
I think we can maybe come up with a better way of encapsulating this automatic link open outside of still using this no-render `LinkButton`. Perhaps something like a custom hook like `useAutomaticLinkOpen` or something of the like instead of having a no-render component.

Reply:
alternatively, we can just rename `LinkButton` to something more appropriate since it's no longer a button

Reply:
renamed it LaunchLink